release-notes/v3.12.0-release-notes.md

```diff
@@ -0,0 +1,56 @@
+27 Jan 2020
+
+#### OpenShift v4.2 support with operator
+
+Calico now supports the latest major release of OpenShift! Included in this
+Calico release is an operator that installs and manages your Calico cluster on
+OpenShift. For more information, see the OpenShift v4 [installation guide](https://docs.projectcalico.org/v3.12/getting-started/openshift/installation).
+
+#### IPv6 service advertisement
+
+If you are running IPv6 or dual-stack Calico, I have exciting news for you: Calico now supports
+advertising IPv6 service clusterIPs over BGP in addition to IPv4 service IPs! To advertise your IPv6 services,
+simply add the IPv6 clusterIP cidrs of your workloads to your default `BGPConfiguration`. For more information,
+see the [howto](https://docs.projectcalico.org/v3.12/networking/advertise-service-ips) guide.
```

Comment:
Instead of "See the howto guide" I would say "See the [documentation for advertising service IPs](...)"